Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator.hs

```diff
@@ -105,7 +105,8 @@ data SendPDNNotificationToDriverJobData = SendPDNNotificationToDriverJobData
   { startTime :: UTCTime,
     endTime :: UTCTime,
     merchantId :: Id DM.Merchant,
-    merchantOperatingCityId :: Maybe (Id DMOC.MerchantOperatingCity)
+    merchantOperatingCityId :: Maybe (Id DMOC.MerchantOperatingCity),
+    retryCount :: Int
```

Comment:
Backward incompatible, it should be Maybe else old job will not be picked up.

Reply:
Sure